### ISSUE TITLE
Sample dataset?

### ISSUE THREAD

**Issue by boxabirds** [WelkinYang/Learn2Sing2.0#6]
Hi -- this is a pretty amazing project. I understand if you can't share all of your data set but I wondered if you could provide a small sample with the precise format and structure that could help others wanting to create their own, and guidance on what tools you used to annotate the source recordings -- taking 5 hours of songs and marking them up at phone level is a huge task indeed.

**Comment by yosun** [comment]
Yes, if you could give more tips on generating the sample data that would be appreciated. 

Also, ideally, a one shot converter to sample set would be much more appreciated. ^.~
